src/gatling/scala/uk/gov/hmcts/reform/draftstore/actions/setup/LeaseServiceToken.scala

```diff
@@ -23,7 +23,7 @@ object LeaseServiceToken {
     feed(serviceNameFeeder)
       .exec(
         http("Lease service token")
-          .post(url)
+          .post(s2sUrl + "/testing-support/lease")
```

Comment:
What do you think about [interpolating](https://docs.scala-lang.org/overviews/core/string-interpolation.html) that string instead?

Reply:
I use it in other places, prefer '+' here.